modules/microshift-olm-deploy-ops-global-ns.adoc

```diff
@@ -0,0 +1,122 @@
+//Module included in the following assemblies:
+//
+//* microshift_running_apps/microshift-operators-olm.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-OLM-deploy-Operators_{context}"]
+= Adding OLM-based Operators to a networked cluster using the global namespace
+
+Use this procedure if you want to deploy different operators to different namespaces. For {microshift-short} clusters that have network connectivity, Operator Lifecycle Manager (OLM) can access sources hosted on remote registries. The following procedure lists the basic steps of using configuration files to install an Operator that uses the global namespace.
+
+[NOTE]
+====
+If you want to use an Operator installed in a different namespace, or in more than one namespace, both the catalog source and the Subscription CR that references the Operator must be running in the `openshift-marketplace` namespace.
+====
+
+.Prerequisites
+* The {oc-first} is installed.
+* Operator Lifecycle Manager (OLM) is installed.
+* You have created a custom catalog in the global namespace.
+
+.Procedure
+
+. Confirm that OLM is running by using the following command:
++
+[source,terminal]
+----
+$ oc -n openshift-operator-lifecycle-manager get pod -l app=olm-operator
+----
++
+.Example output
+[source,terminal]
+----
+NAME                            READY   STATUS    RESTARTS   AGE
+olm-operator-66c97f894f-55wp2   1/1     Running   0          126m
+----
+
+. Confirm that the OLM catalog Operator is running by using the following command:
++
+[source,terminal]
+----
+$ oc -n openshift-operator-lifecycle-manager get pod -l app=catalog-operator
+----
++
+.Example output
+[source,terminal]
+----
+NAME                                READY   STATUS    RESTARTS   AGE
+catalog-operator-7cf77c459d-jbrd7   1/1     Running   0          126m
+----
+
+//Q which directory do we place these YAMLs in? do we need to specify?
```

Comment:
it doesn't matter, they're used directly in the `oc apply` command which is executed by the user, microshift doesn't do anything with it (unless they're packaged as RPM, but that's another topic)